### PR TITLE
implement naming schema for http plugins

### PR DESCRIPTION
**Pull request by rarguelloF**
### What does this PR do?

Update the following plugins to use naming schema:
- `http` (client and server)
- `http2` (client and server)
- `next` (server)
- `fetch` (client)
- `aws-sdk`: 
  - uses operation names as follows (depending on the AWS operation)
    - `sqs`: `aws.sqs.send`, `aws.sqs.process` or `aws.sqs.request`
    - `sns`: `aws.sns.send` or `aws.sns.request`
    - `lambda`: `aws.lambda.invoke` or `aws.lambda.request`
    - rest: `aws.{{ AWS_SERVICE }}.request`
 
### Motivation

Consistent span and service names.

### Plugin Checklist

- [x] Unit tests.
- [ ] TypeScript [definitions][1].
- [ ] TypeScript [tests][2].
- [ ] API [documentation][3].
- [ ] CircleCI [jobs/workflows][4].
- [ ] Plugin is [exported][5].

[1]: https://github.com/DataDog/dd-trace-js/blob/master/index.d.ts
[2]: https://github.com/DataDog/dd-trace-js/blob/master/docs/test.ts
[3]: https://github.com/DataDog/documentation/blob/master/content/en/tracing/trace_collection/library_config/nodejs.md
[4]: https://github.com/DataDog/dd-trace-js/blob/master/.circleci/config.yml
[5]: https://github.com/DataDog/dd-trace-js/blob/master/packages/dd-trace/src/plugins/index.js

### CODE REVIEW

**Review by jbertran — APPROVED** [review]
LGTM! Added a commit to make testing data consistent with other plugins, the rest is good to go.

File: packages/dd-trace/src/service-naming/schemas/util.js [hunk 1]
```diff
@@ -2,4 +2,22 @@ function identityService ({ tracerService }) {
   return tracerService
 }
 
-module.exports = { identityService }
+function getFormattedHostString ({ host, port }) {
```
Comment (tlhunter) [inline packages/dd-trace/src/service-naming/schemas/util.js:5]: It it intentional that `getFormattedHostString ({ port:123 })` returns `'123'`?

Comment (tlhunter) [reply packages/dd-trace/src/service-naming/schemas/util.js:5]: Eh, probably a non-issue

**Review by tlhunter — APPROVED** [review]

File: packages/dd-trace/src/service-naming/schemas/util.js (hunk 1 above)
Comment (jbertran) [reply packages/dd-trace/src/service-naming/schemas/util.js:5]: We're relying on sessionDetails always having a host in http / http2, which sounds like a fair assumption to me.

**Pull request merged**